src/components/Header.js

```diff
@@ -1,10 +1,64 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ user, profileImg, changeUser }) {
+  const handleButtonClick = () => {
+    changeUser();
+  };
+  return (
+    <Wrapper>
+      <button onClick={handleButtonClick}>
+        <img src={profileImg} alt="userProfile"/>
+      </button>
```

Comment:
이부분도 styled-components를 사용하시면 좋을 것 같아요 아래 코드도 참고해주세요..!

```suggestion
      <Button onClick={changeUser}>
        <ProfileImg src={profileImg} alt="userProfile"/>
      </Button>
```